recipes/systemc/2.3.3/conanfile.py

```diff
@@ -0,0 +1,90 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class SystemcConan(ConanFile):
+    name = "systemc"
+    version = "2.3.3"
+    description = """SystemC is a set of C++ classes and macros which provide
+                     an event-driven simulation interface."""
+    homepage = "https://www.accellera.org/"
+    url = "https://github.com/conan-io/conan-center-index "
+    license = "MIT"
+    topics = ("simulation", "modeling", "esl", "tlm")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "disable_async_updates": [True, False],
+        "disable_copyright_msg": [True, False],
+        "disable_virtual_bind": [True, False],
+        "enable_assertions": [True, False],
+        "enable_immediate_self_notifications": [True, False],
+        "enable_pthreads": [True, False]
+    }
+    default_options = {
+        "shared": True,
+        "disable_async_updates": False,
+        "disable_copyright_msg": False,
+        "disable_virtual_bind":  False,
+        "enable_assertions": True,
+        "enable_immediate_self_notifications": False,
+        "enable_pthreads": False
+    }
+    generators = "cmake"
+    exports_sources = "sc_string_view.patch"
```

Comment:
```suggestion
    exports_sources = "patches/**"
```
It is custom in this repo to store the patches in a `patches` subdirectory. (the `conandata.yml` files stores the list of patches)